ipi/engine/motion/dynamics.py

```diff
@@ -607,7 +607,7 @@ class NPTIntegrator(NVTIntegrator):
     def pstep(self, level=0):
         """Velocity Verlet monemtum propagator."""
 
-        if np.allclose(self.forces.vir, np.zeros(len(self.forces.vir))):
+        if np.array_equiv(self.forces.vir, np.zeros(len(self.forces.vir))):
```

Comment:
is this still required if we are using the dummy driver?

Reply:
This actually covers a safety net that @mahrossi put in place for ppl doing NPT without activating the calculation of the stress. I was bugged that this broke the ideal gas driver, so I had gas return 1e-100 and made these check for *exactly* zero